### DISCUSSION TITLE
When are my models loaded ?

### DISCUSSION THREAD

**Discussion by uzair004** [vladmandic/human#161] [Q&A] [answered]
I am using Node demo file to learn and explore this library.

It looks like every time i upload an image , all models gets loaded and perform operations.

## How much performance improvement it will bring ?

**Comment by vladmandic** [comment, marked as answer]
models are always loaded only once for the entire lifespan of the `human` object  
only way that models get loaded again and again is if you're creating human object again and again  
(e.g., `const human = new Human();`)

few more notes:
- models can be loaded explicitly with `await human.load()` or they will be loaded automatically during first usage
- only models enabled in configuration get loaded, human does not load everything by default

**Reply by uzair004** [reply]
Get it, Thanks
